### DISCUSSION TITLE
How to port local staging code to production on server?

### DISCUSSION THREAD

**Discussion by biapar** [strapi/strapi#7230] [Legacy Topics]
How to port local staging code to production on server?
Is better to copy the all source folders and then build on server?
Where is located the NOSQL file?

**Comment by lauriejim** [comment]
Hello! How do you do to push your code from development to staging?

**Reply by biapar** [reply]
When I use Vue.JS or similar by copy build/dist folder.

**Reply by lauriejim** [reply]
I'm talking about your Strapi app, not your frontend application.
I suppose your are using GitHub to push your app.
Well you will use the same principal to push your app in production.

**Reply by biapar** [reply]
I don't have the Strapi application under Github, now. So my question is, How to manually do?

**Reply by lauriejim** [reply]
I don't know. You can probably connect via ftp and copy past your files.
